src/common/segment-utils.ts

```diff
@@ -104,3 +105,63 @@ type FacebookBasicEvent =
 export const trackFacebookBasicEvent = (eventName: FacebookBasicEvent) => {
   analytics.track(eventName, {}, { integrations: { All: false, 'Facebook Pixel': true } });
 };
+
+/**
+ * Register a Segment plugin that will parse url paths from learn.xip.co.
+ *
+ * @see https://github.com/segmentio/analytics-next/tree/master/packages/browser#-plugins
+ *
+ */
+export const urlPathFilter: Plugin = {
+  name: 'Clickfunnels Path Filter',
+  type: 'enrichment',
+  version: '1.0.0',
+
+  isLoaded: () => true,
+  load: () => Promise.resolve(),
+
+  track: (ctx) => {
+    const url = new URL(ctx.event.properties?.url || '');
```

Comment:
Changed to window.location.href. I think this would work.

Reply:
jk I was looking at the wrong spot. Keeping this as ctx.event.properties since it's within the context of the plugin, which I want to only deal with event data